test/sdf/pose_1_9.sdf

```diff
@@ -0,0 +1,163 @@
+<?xml version="1.0" ?>
+<sdf version="1.9">
+  <world name="default">
+    
+    <model name="model_with_empty_pose">
+      <pose></pose>
+      <link name="link"/>
+    </model>
+    
+    <model name="model_with_pose_value">
+      <pose>1 2 3 1 2 3</pose>
+      <link name="link"/>
+    </model>
+    
+    <model name="model_with_empty_translation">
+      <pose>
+        <translation></translation>
+      </pose>
+      <link name="link"/>
+    </model>
+    
+    <model name="model_with_translation_value">
+      <pose>
+        <translation>2 3 4</translation>
+      </pose>
+      <link name="link"/>
+    </model>
+
+    <model name="model_with_empty_rpy_deg">
+      <pose>
+        <rotation type="rpy_degrees"></rotation>
+      </pose>
+      <link name="link"/>
+    </model>
+
+    <model name="model_with_rpy_deg_value">
+      <pose>
+        <rotation type="rpy_degrees">0 90 180</rotation>
+      </pose>
+      <link name="link"/>
+    </model>
+
+    <model name="model_with_empty_rpy_rad">
+      <pose>
+        <rotation type="rpy_radians"></rotation>
+      </pose>
+      <link name="link"/>
+    </model>
+
+    <model name="model_with_rpy_rad_value">
+      <pose>
+        <rotation type="rpy_radians">1 2 3</rotation>
```

Comment:
I would be careful using euler angles larger than pi/2 in an example because I believe they will be truncated internally when loaded and then show a different value when printed out again. For example, using the `quaternion_*_euler` executables in the ign-math examples:

~~~
$ ./quaternion_from_euler 1 2 3
Converting Euler angles:
 roll   1.000000 radians
 pitch  2.000000 radians
 yaw    3.000000 radians

 roll     57.295780 degrees
 pitch   114.591559 degrees
 yaw     171.887339 degrees

to Quaternion
 W  0.435953
 X -0.718287
 Y  0.310622
 Z  0.444435

to Rotation matrix
    0.411982  -0.833738  -0.367630
   -0.058727  -0.426918   0.902382
   -0.909297  -0.350175  -0.224845
~~~

then converting the quaternion back to Euler:

~~~
$ ./quaternion_to_euler 0.435953 -0.718287 0.310622 0.444435
Normalizing Quaternion components:
  W 0.435953
  X -0.718287
  Y 0.310622
  Z 0.444435
to
  W 0.435953
  X -0.718287
  Y 0.310622
  Z 0.444435

Converting to Euler angles
 roll  -2.141591 radians
 pitch  1.141592 radians
 yaw   -0.141591 radians

 roll  -122.704130 degrees
 pitch  65.408409 degrees
 yaw   -8.112557 degrees

to Rotation matrix
    0.411983  -0.833737  -0.367631
   -0.058726  -0.426918   0.902382
   -0.909297  -0.350176  -0.224845
~~~

the rotation matrices are the same, but it results in different Euler angles